views/widgets/Full_Width_Slider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/* @var $image string */
+/* @var $instance array */
+?>
+
+
+<div class="pb-widget-inner">
```

Comment:
this class is inserted automatically with $instance['before_widget'], (like usual WP widget)

Reply:
removed